collection.html

```diff
@@ -0,0 +1,83 @@
+<!DOCTYPE html>
```

Comment:
Is this an extra file that should be deleted? the `pages/collections.html` is the one we're using I believe?

Reply:
yep, good catch. I can remove that